docs/design/features/host-error-codes.md

```diff
@@ -8,45 +8,42 @@ Note that the exit code returned by running an application via `dotnet.exe` or `
-| `CoreHostEntryPointFailure`           | `0x80008084` | `-2147450748`          | `132`         | One of the dependent libraries is missing a required entry point. |
-| `CoreHostCurHostFindFailure`          | `0x80008085` | `-2147450747`          | `133`         | If the hosting component is trying to use the path to the current module (the hosting component itself) and from it deduce the location of the installation. Either the location of the current module could not be determined (some weird OS call failure) or the location is not in the right place relative to other expected components. For example the `hostfxr` may look at its location and try to deduce the location of the `shared` folder with the framework from it. It assumes the typical install layout on disk. If this doesn't work, this error will be returned. |
-| `CoreClrResolveFailure`               | `0x80008087` | `-2147450745`          | `135`         | If the `coreclr` library could not be found. The hosting layer (`hostpolicy`) looks for `coreclr` library either next to the app itself (for self-contained) or in the root framework (for framework-dependent). This search can be done purely by looking at disk or more commonly by looking into the respective `.deps.json`. If the `coreclr` library is missing in `.deps.json` or it's there but doesn't exist on disk, this error is returned. |
-| `CoreClrBindFailure`                  | `0x80008088` | `-2147450744`          | `136`         | The loaded `coreclr` library doesn't have one of the required entry points. |
-| `CoreClrInitFailure`                  | `0x80008089` | `-2147450743`          | `137`         | The call to `coreclr_initialize` failed. The actual error returned by `coreclr` is reported in the error message. |
-| `CoreClrExeFailure`                   | `0x8000808a` | `-2147450742`          | `138`         | The call to `coreclr_execute_assembly` failed. Note that this does not mean anything about the app's exit code, this failure occurs if `coreclr` failed to run the app itself. |
+| `InvalidArgFailure`                   | `0x80008081` | `-2147450751`          | `129`         | One or more arguments are invalid. |
+| `CoreHostLibLoadFailure`              | `0x80008082` | `-2147450750`          | `130`         | Failed to load a [hosting component](./host-components.md). If loading a hosting compnent via `LoadLibrary`/`dlopen` fails, this error code is returned. The most common case for this failure is if the library is missing some of its dependencies (for example the necessary CRT is missing on the machine), likely corrupt or incomplete install. This error code is also returned from `corehost_resolve_component_dependencies` if it's called on a `hostpolicy` which has not been initialized via the hosting layer. This would typically happen if `coreclr` is loaded directly without the hosting layer and then `AssemblyDependencyResolver` is used (which is an unsupported scenario). |
```

Comment:
```suggestion
| `CoreHostLibLoadFailure`              | `0x80008082` | `-2147450750`          | `130`         | Failed to load a [hosting component](./host-components.md). If loading a hosting component via `LoadLibrary`/`dlopen` fails, this error code is returned. The most common case for this failure is if the library is missing some of its dependencies (for example the necessary CRT is missing on the machine), likely corrupt or incomplete install. This error code is also returned from `corehost_resolve_component_dependencies` if it's called on a `hostpolicy` which has not been initialized via the hosting layer. This would typically happen if `coreclr` is loaded directly without the hosting layer and then `AssemblyDependencyResolver` is used (which is an unsupported scenario). |
```